adapters/MythicMobsAdapter/src/main/kotlin/com/caleb/typewriter/mythicmobs/entries/fact/MobExsitFact.kt

```diff
@@ -0,0 +1,33 @@
+package com.caleb.typewriter.mythicmobs.entries.fact
+
+import io.lumine.mythic.bukkit.MythicBukkit
+import lirand.api.extensions.server.server
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.entries.ReadableFactEntry
+import me.gabber235.typewriter.facts.Fact
+import me.gabber235.typewriter.utils.Icons
+import java.util.*
+
+@Entry("mob_exist_fact", "If the mob exist in the world", Colors.PURPLE, Icons.PLACE_OF_WORSHIP)
+data class MobExsitFact(
+	override val id: String = "",
+	override val name: String = "",
+	override val comment: String = "",
+	@Help("The mob's name to check")
+	val mob: String = "",
+) : ReadableFactEntry {
+	override fun read(playerId: UUID): Fact {
+		val player = server.getPlayer(playerId) ?: return Fact(id, 0)
+		val mob = MythicBukkit.inst().mobManager.getMythicMob(mob) ?: return Fact(id, 0)
+		var value = 0
+		for (activeMob in MythicBukkit.inst().mobManager.activeMobs) {
+			if (!activeMob.name.equals(mob)) continue;
+			value = 1
+			break
+		}
+
+		return Fact(id, value)
```

Comment:
This is a mess. You check if the player exists, but do nothing with it. You try to compare a `Optional<MythicMob>` to a `String` on line `26`. 


Please change this Fact from an existence to a count. Such that the value of the fact is the count of the mobs. This way, one has more flexibility to check how many mobs of the same type/name exists. So if you have a `Custom Zombie` that you can check how many `Custom Zombies` there are.